suite-native/qr-code/src/components/AddressQRCode.tsx

```diff
@@ -49,17 +48,17 @@ export const AddressQRCode = ({ address, backgroundElevation = '0' }: AddressQRC
                     size="small"
                     iconLeft="copy"
                     onPress={handleCopyAddress}
-                    colorScheme={`tertiaryElevation${backgroundElevation}`}
+                    colorScheme="tertiaryElevation1"
                 >
-                    Copy
+                    {translate('moduleReceive.receiveAddressCard.qrCode.copy.button')}
```

Comment:
This strings are not part of `moduleReceive` but they are universal to package `qr-code`

Reply:
fixed in https://github.com/trezor/trezor-suite/commit/8af235d6e7bb7d75e4232baa31ce1ffe754a15b6